frontend/spec/controllers/spree/home_controller_spec.rb

```diff
@@ -1,14 +1,6 @@
 require 'spec_helper'
 
 describe Spree::HomeController, type: :controller do
-  it 'provides current user to the searcher class' do
-    user = mock_model(Spree.user_class, last_incomplete_spree_order: nil, spree_api_key: 'fake')
-    allow(controller).to receive_messages try_spree_current_user: user
-    expect_any_instance_of(Spree::Config.searcher_class).to receive(:current_user=).with(user)
```

Comment:
We don't initialize search in the `HomeController` anymore, so these specs can be deleted